src/domain/user-management.js

```diff
@@ -1,9 +1,9 @@
-export const CREATE_USER_RESULT_SUCCESS = 'create-user-result-success';
-export const CREATE_USER_RESULT_DUPLICATE_EMAIL = 'create-user-result-duplicate-email';
-export const CREATE_USER_RESULT_DUPLICATE_USERNAME = 'create-user-result-duplicate-username';
+export const SAVE_USER_RESULT_SUCCESS = 'save-user-result-success';
```

Comment:
Would it make sense to change these to 
export const SAVE_USER_RESULT = {
   success: 'save-user-result-success'
  ...
}

I used this in the image component as per airbnb rules :).